include/libzfs.h

```diff
@@ -441,8 +441,10 @@ extern int zpool_events_next(libzfs_handle_t *, nvlist_t **, int *, unsigned,
     int);
 extern int zpool_events_clear(libzfs_handle_t *, int *);
 extern int zpool_events_seek(libzfs_handle_t *, uint64_t, int);
+extern void zpool_obj_to_path_impl(zpool_handle_t *, uint64_t, uint64_t, char *,
+    size_t, boolean_t);
```

Comment:
It'd be nice not to include an `_impl` function in this public header.  How about keeping the `_impl` function private and adding a new public wrapper function for this called something like `zpool_obj_to_path_ds(pool_handle_t *, uint64_t, uint64_t, char *, size_t)`.